### PR TITLE
Wording update

### PR DESCRIPTION
**Pull request by meis**
- Free t-shirt announcement
  -Link to approved talks from 'submit a talk' widget

### CODE REVIEW

File: actdocs/templates/index/description [hunk 1]
```diff
@@ -9,9 +9,12 @@
       Oh! and it's <b>free</b> to attend! You just need to sign in with your Act user
       (or create a new one if you don't have one) and register for the conference.
       There are limited places, so better hurry up!
+      (Everyone registered before the 20th of October will also get a <b>free t-shirt</b>)
```
Comment (diegok) [inline actdocs/templates/index/description:12]: What do you think of something more like: `Make sure you get the conference t-shirt of your size by registering before October 21th.`

Comment (meis) [reply actdocs/templates/index/description:12]: It's ok for me. Can you please do the change? I'll be off the laptop most of today...

Comment (jluis) [reply actdocs/templates/index/description:12]: Better:
(Remember to register and notify  your size & style (men/women) before the 20th of October and your <b>free t-shirt</b> will fit)

**Comment by meis** [comment]
I guess that's no longer relevant...

**Pull request closed without merging**